woocommerce/woocommerce-ios
# [Design review] Products tab: divider updates

Closes: #12212


## Description


- Added function removeShadow() to remove shadow from navigation bar
- Added separator line between toolbar and tableview on products list screen

## Testing instructions


- Open products tab on both iPhone and iPad
- Check that there **IS NOT** divider below "Products" navigation title and check that there **IS** a divider between sorting toolbar and table view

## Screenshots


| Before      | After |
| ----------- | ----------- |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-03-18 at 10 40 47](https://github.com/woocommerce/woocommerce-ios/assets/6242034/56321b99-e341-4e7b-b618-58008f4d35d2)      | ![Simulator Screenshot - iPhone 15 Pro - 2024-03-18 at 10 35 12](https://github.com/woocommerce/woocommerce-ios/assets/6242034/07343437-dfe4-4e93-ae95-69b59badd53d)       |

| Before      | After |
| ----------- | ----------- |
| ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-03-18 at 10 42 08](https://github.com/woocommerce/woocommerce-ios/assets/6242034/6ca0bc1b-72ad-4536-859a-b34d8a19cc92)      | ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-03-18 at 10 34 20](https://github.com/woocommerce/woocommerce-ios/assets/6242034/7943bf4a-7087-4f4d-bbd5-2ed606c5fbaa)       |

---
